crates/torii/graphql/src/object/metadata/mod.rs

```diff
@@ -121,8 +122,8 @@ fn metadata_connection_output(
                 extract_str_mapping("name", &serde_value),
                 extract_str_mapping("description", &serde_value),
                 extract_str_mapping("website", &serde_value),
-                extract_str_mapping("icon_uri", &serde_value),
-                extract_str_mapping("cover_uri", &serde_value),
```

Comment:
I believe these needs to remain snake_case, we store the ipfs content as json str in db and the name format for content are all snake_case

Reply:
Makes sense 👍